src/components/CardEditor/CardGalleryList/CardGalleryList.jsx

```diff
@@ -78,8 +84,12 @@ const CardGalleryList = ({ supportedCardTypes, onAddCard, i18n }) => {
       items={supportedCardTypes.map((cardType) => ({
         id: cardType,
         content: {
-          value: mergedI18n[`cardType_${cardType}`] || cardType,
-          icon: iconTypeMap[cardType],
+          value: mergedI18n[cardType] || cardType,
+          icon: (
+            <div className={`${iotPrefix}--card-gallery-list__icon`}>
+              {mergedIcons[cardType] || <Analyze />}
```

Comment:
I think it's okay to mirror how we had it before and just use `mergedIcons[cardType]` without the `Analyze` fallback. 
```suggestion
              {mergedIcons[cardType]}
```